src/main/resources/templates/clojure-lsp/clientSettings.json

```diff
@@ -1,5 +1,6 @@
 {
   "caseSensitive": true,
+  "lineCommentPrefix": ";",
```

Comment:
Updated the bundled language server template client configurations for the respective languages' line/block comments and statement terminators.